src/pyDE1/scale/felicita_arc.py

```diff
@@ -0,0 +1,147 @@
+"""
+Copyright © 2022 Mimoja. All Rights Reserved.
+
+License for this software, part of the pyDE1 package, is granted under
+GNU General Public License v3.0 only
+SPDX-License-Identifier: GPL-3.0-only
+"""
+
+import enum
+import sys
+import time
+from typing import Callable
+
+import pyDE1
+from pyDE1.scale import Scale
+from pyDE1.scale.events import ScaleWeightUpdate
+
+logger = pyDE1.getLogger('Scale.FelicitaArc')
+
+class FelicitaArc(Scale):
+
+    def __init__(self):
+        super(FelicitaArc, self).__init__()
+        self._nominal_period = 0.1  # seconds per sample
+        self._minimum_tare_request_interval = 2.5 * self._nominal_period # Guessed, not tested
+        self._sensor_lag = 0.45  # seconds, including all transit delays
+        self._tare_timeout = 1.0  # seconds until considered coincidence
+        self._tare_threshold = 0.05  # grams, within this, considered "at zero"
+
+        # Enable tare on button 1, hold UUID if need to unsubscribe
```

Comment:
If no buttons supported, these two can be dropped.

---

Does the battery report on the standard CUUID or is it somewhere else?

It seems to be that, **_in the future,_** it would be valuable to provide battery level for this, the Acaias, and other scales that reasonably run off batteries, to the APIs.